DOCUMENTATION_WRITING.md

```diff
@@ -0,0 +1,54 @@
+# How to Write Documentation
+
+This document explains how to write documentation such as tutorials, guides or recommendations to be hosted in ocaml.org.
+
+## Materials
+
+- Reference. This includes the [OCaml Manual](https://ocaml.org/releases/latest/manual.html), the [Opam manual](https://opam.ocaml.org/) and [Dune's documentation](https://dune.readthedocs.io/en/stable/). This is considered upstream material, refer to the corresponding projects to propose changes in them.
+- Tutorials
+- Guides
+- Recommended practices
+
+## Audience
+
+Not our audience:
+* Students going to universities
+* Absolutely clueless about programming, we do not teach from scratch
+
+Our audience:
+* Self-directed learners without a tutor
+* Already know some programming basics (one other programming programming)
+* People who want to learn untainted functional programming (have seen functional patterns in other languages and are curious about “the real thing”)
+* Likely a majority of consumers of libraries, hopefully some authors or people who turn into it
+* English level B2 ideally
+
+## Goals
+
+1. Enable people to use OCaml for real projects / at the job / side projects / Advent of Code
+1. Get people who want to build and contribute to the ecosystem up to speed and building something good quickly
+1. Help people learn untainted FP through learning a language
```

Comment:
```suggestion
1. Help people learn untainted FP through learning a language
```

I disagree with this goal and the definition of untainted FP is not clear either.